test/libyul/yulOptimizerTests/fullSuite/proxy.yul

```diff
@@ -0,0 +1,59 @@
+{
+    mstore(64, 128)
+    if callvalue() { revert(0, 0) }
+    let _1 := 0
+    calldatacopy(128, _1, calldatasize())
+    mstore(add(128, calldatasize()), _1)
+    pop(delegatecall(gas(), 25, 128, calldatasize(), _1, _1))
+    let data := _1
+    switch returndatasize()
+    case 0 { data := 96 }
+    default {
+        let result := and(add(returndatasize(), 63), not(31))
+        let memPtr := mload(64)
```

Comment:
Example from https://github.com/ethereum/solidity/issues/10645. This `mload(64)` can be resolved.